eslint.config.mjs

```diff
@@ -119,6 +121,7 @@ export default [
       radix: ['error', 'as-needed'],
       'react/prop-types': 0,
       'react/display-name': 0,
+      'react-compiler/react-compiler': 'warn',
```

Comment:
I assume at some point we want to change this to error out when we move to react 19? @tlabaj Do we need a story to revisit this in the future.

Reply:
Already logged it, see #11138 😉

Reply:
The `error` value can be set once all warnings are fixed, no need to wait for React 19. It only notifies you of patterns that are incompatible with the React Compiler, but does not introduce anything incompatible.